.pre-commit-config.yaml

```diff
@@ -1,3 +1,23 @@
+# Enable automatic updates via Github Actions as well.
+# See https://pre-commit.ci
+ci:
+    # Let pre-commit.ci automatically update PRs with formatting fixes.
+    autofix_prs: true
```

Comment:
There's an example of how this works in this PR's commit history in case you're curious - just a simple isort fixup.